src/test/java/school/hei/haapi/integration/GroupIT.java

```diff
@@ -155,7 +155,7 @@ void manager_write_create_ok() throws ApiException {
     TeachingApi api = new TeachingApi(manager1Client);
     List<Group> created = api.createOrUpdateGroups(List.of(toCreate3, toCreate4));
     List<Group> createdWithStudent = api.createOrUpdateGroups(List.of(toCreate5));
-    List<Student> students = api.getAllStudentByGroup(createdWithStudent.get(0).getId(), 1, 10);
+    List<Student> students = api.getAllStudentByGroup(createdWithStudent.get(0).getId(), 1, 10, null);
```

Comment:
you might want to change your tests with real values in order to do some TDD but for now, you can merge this after resolving the comments